suricata/update/main.py

```diff
@@ -1018,6 +1019,20 @@ def check_output_directory(output_dir):
                 "Failed to create directory %s: %s" % (
                     output_dir, err))
 
+
+def show_defaults():
+    row_format = "{:<30}" * 2
+    for conf_key, conf_val in config._config.items():
+        if conf_val and isinstance(conf_val, Iterable) and \
+                not isinstance(conf_val, str):
+            print(conf_key)
+            if isinstance(conf_val, list):
+                for i in conf_val:
+                    print("{:>30}{}".format("", i))
+        else:
+            print(row_format.format(conf_key, conf_val if conf_val else "Not set"))
```

Comment:
hmm this should not have the internal information like "subcommand". Needs a spin.

Reply:
On second thoughts, this is probably a redundant feature. `-v` gives most of the required information and now we even have an `--offline` option in case we do not want to update the rules while getting this info.

Reply:
I could live without this feature.  But its not a bad feature.  Some command with the following output could be useful:
```
Using Suricata: /usr/bin/suricata
Using Suricata configuration file: /etc/suricata/suricata.yaml
Config files Suricata-update would load...
Writing rules to ....
```
Thats pretty rough, but gets the idea across.

Another idea would be a `--dry-run` that goes through all the motions, but doesn't actually do anything. Could be tricky though.

Reply:
Should the `--dry-run` feature be implemented instead then?